client/src/hooks/useOfflineQueueListener.ts

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from 'react';
+import { useApolloOfflineClient } from 'react-offix-hooks';
+import { useGetOfflineStore } from './useGetOfflineStore';
+
+export const useOfflineQueueListener = () => {
```

Comment:
So this seems like a lot of code just to find out the length of the offline queue. We used to do something similar in the old showcase app but it's no longer needed because you can actually directly access the offline queue on `client.queue`.

This could probably simplify the entire functionality around displaying the items in the offline queue.

Reply:
Ahh there had to be a simpler way of doing that. Good to know.